.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all"
+}
```

Comment:
Let's keep a new line at the end of all files, as mentioned in the POSIX standard.